src/welcome.md

```diff
@@ -4,9 +4,9 @@
 [![GitHub contributors](https://img.shields.io/github/contributors/google/comprehensive-rust?style=flat-square)](https://github.com/google/comprehensive-rust/graphs/contributors)
 [![GitHub stars](https://img.shields.io/github/stars/google/comprehensive-rust?style=flat-square)](https://github.com/google/comprehensive-rust/stargazers)
 
-This is a three day Rust course developed by the Android team. The course covers
+This is a free Rust course developed by the Android team. The course covers
```

Comment:
For a separate PR, but it may also be worth explaining what "the Android team" means.

Reply:
Good point, thanks!